locales/de.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Execute binaries from npm packages.\n%s": "Führt Programme aus, die von npm Paketen bereitgestellt werden.\n%s",
+  "Package to be installed.": "Das zu installierende Paket.",
+  "Location of the npm cache.": "Ort des npm Zwischenspeichers.",
+  "Skip installation if a package is missing.": "Überspringe die Installation falls ein Paket fehlt.",
+  "Path to user npmrc.": "Pfad zur Benuzter_innen npmrc.",
```

Comment:
German is brutally gendered, so for more inclusive writing, we have to workaround with funky punctuation.

Here for the Translation of `user` the underscore-version is a mix version of the male “Benutzer” + female “Benutzerinnen” + [Gender Gap](https://de.wikipedia.org/wiki/Gendergap_(Linguistik))(🇩🇪Wikipedia), where the underscore is the placeholder for all other genders.

An [alternative](https://de.wikipedia.org/wiki/Gendergap_(Linguistik)#Variationen) would be `Benutzer*innen`. The difference is mostly stylistic, although `_` is slightly more common as per my perception.

Also acceptable, although less inclusive (no space/star for non-binaries): `BenutzerInnen`.